app/components/modals/cropper-modal.js

```diff
@@ -5,8 +5,13 @@ export default class extends ModalBase {
     let viewport = {};
     let factor = 150;
     const aspectRatio = this.getWithDefault('aspectRatio', [2, 1]);
-    viewport.width = aspectRatio[0] * factor;
-    viewport.height = aspectRatio[1] * factor;
+    if (this.session.currentRouteName === 'public.cfs.edit-speaker' || this.session.currentRouteName === 'public.cfs.new-speaker') {
+      viewport.width = 300;
+      viewport.height = 300;
```

Comment:
This will crop the image to 300px by 300px

We don't want that, we just want a square image